pkg/server/backgroundsvcs/background_services.go

```diff
@@ -34,7 +35,7 @@ func ProvideBackgroundServiceRegistry(
 	tracing tracing.Tracer, remoteCache *remotecache.RemoteCache, secretsService *secretsManager.SecretsService,
 	// Need to make sure these are initialized, is there a better place to put them?
 	_ *plugindashboards.Service, _ *dashboardsnapshots.Service, _ *pluginsettings.Service,
-	_ *alerting.AlertNotificationService, _ serviceaccounts.Service,
+	_ *alerting.AlertNotificationService, _ serviceaccounts.Service, _ dashboardimport.Service,
```

Comment:
Do we need this here? IE is it actually doing anything? 🤔

Reply:
It won't be included in the dependency graph if not added here and by that not run.

Reply:
Remove it and give it a shot 😉

Reply:
Oh okay, since plugindashboards service now import dashboardimport service it doesn't need to be there. Good catch